### PR TITLE
Migrate and lint Tree View js

### PR DESCRIPTION
**Pull request by bbovenzi**
Continuation of #14115 for Tree View:

- Move nearly all js to its own file
- Lint tree js

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: airflow/www/templates/airflow/tree.html [hunk 1]
```diff
@@ -20,6 +20,11 @@
 {% extends "airflow/dag.html" %}
 {% block page_title %}{{ dag.dag_id }} - Tree - {{ appbuilder.app_name }}{% endblock %}
 
+{% block head_meta %}
+  {{ super() }}
+  <meta name="blur" content="{{ blur }}">
```
Comment (ryanahamilton) [inline airflow/www/templates/airflow/tree.html:25]: We should really change this variable name to something more descriptive like `is_demo_mode` (in a separate PR).

File: airflow/www/templates/airflow/tree.html [hunk 2]
```diff
@@ -87,376 +92,8 @@
   {{ super() }}
   <script src="{{ url_for_asset('d3.min.js') }}"></script>
   <script src="{{ url_for_asset('d3-tip.js') }}"></script>
-  <script src="{{ url_for_asset('taskInstances.js') }}"></script>
```
Comment (ryanahamilton) [inline airflow/www/templates/airflow/tree.html:90]: Was this script removed on purpose?

Comment (bbovenzi) [reply airflow/www/templates/airflow/tree.html:90]: Yes. It's being imported into `tree.js` instead.

Comment (ryanahamilton) [reply airflow/www/templates/airflow/tree.html:90]: Ah, I was skimming for `taskInstances`  not `task_instances`.

**Review by ashb — APPROVED** [review]

**Review by ryanahamilton — APPROVED** [review]

**Pull request merged**